XUnitTests/UnitTestGCD.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using GrandCommonDivisor.Core;
+using Xunit;
+
+namespace MyFirstUnitTests
+{
+    public class UnitTestGCD
+    {
+        [Theory]
+        [InlineData(1, 2, 1)]
+        [InlineData(-4, -6, 2)]
+        [InlineData(-2, 2, 2)]
+        public void CalculateGCDGoodData_Passing(int value1, int value2, int expected)
```

Comment:
You don't need to name your tests "Passing" if they are all passing. Usually I use something like "CalculateGCD_WhenValidValuesPassed_ReturnsValidResult".